src/tests/Myr_random.test.js

```diff
@@ -0,0 +1,125 @@
+import Myr from "../myr/Myr";
+
+const myr = new Myr();
+
+describe("Random Function Tests", () => {
+    
+    //randomInt
+    it("randomInt should move box to random x coordinate", () => {
+        let xPos = myr.randomInt();
+        myr.setPosition(xPos, 1, 1);
+        expect(myr.cursor.position).toEqual({ x: xPos, y: 1, z: 1 });
+    });
+
+    it("randomInt value should be within default range", () => {
+        let within = 0, xPos = myr.randomInt();
+        xPos = myr.randomInt();
+        myr.setPosition(1, 1, xPos);
+        if(xPos >= -40 && xPos < 40) {
+            within = 1;
+        }
+        expect(within).toEqual(1);
+    });
+
+    it("random value should be within preset range", () => {
+        let within = 0, xPos = myr.random(15, 20);
+        xPos = Math.floor(xPos);
+        myr.setPosition(1, 1, xPos);
+        if(xPos >= 15 && xPos < 20){
+            within = 1;
+        }
+        expect(within).toEqual(1);
+    });
+
+    //random
+    it("random should move box to random z coordinate", () => {
+        let zPos = myr.random();
+        zPos = Math.floor(zPos);

```

Comment:
Why are you using random and then calling Math.floor instead of randomInt? Again, what is this actually testing?